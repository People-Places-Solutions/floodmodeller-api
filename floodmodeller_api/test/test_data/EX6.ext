[QTBDY__P0000]
X=102678.00714
Y=188968.005016546

[JUNCTION_OPEN_P0000]
X=102678.00714
Y=189860.176880785

[SLUICE_VERTICAL_P0000a]
X=102678.00714
Y=190752.348745024

[HTBDY__P0000b]
X=102678.00714
Y=191644.520609262

[RIVER_SECTION_P0000c]
X=102678.00714
Y=192536.692473501

[RIVER_SECTION_P1000]
X=102678.00714
Y=193428.864337739

[RIVER_SECTION_P2000]
X=102678.00714
Y=194321.036201978

[RIVER_SECTION_P3000]
X=102678.00714
Y=195213.208066217

[RIVER_SECTION_P4000]
X=102678.00714
Y=196105.379930455

[RIVER_SECTION_P5000]
X=102678.00714
Y=196997.551794694

[RIVER_SECTION_P6000]
X=102678.00714
Y=197889.723658932

[RIVER_SECTION_P7000]
X=102678.00714
Y=198781.895523171

[RIVER_SECTION_P8000]
X=102678.00714
Y=199674.06738741

[RIVER_SECTION_P9000]
X=102678.00714
Y=200566.239251648

[RIVER_SECTION_P10000]
X=102678.00714
Y=201458.411115887

[RIVER_SECTION_P11000]
X=102678.00714
Y=202350.582980126

[RIVER_SECTION_P11951A]
X=102678.00714
Y=203242.754844364

[JUNCTION_OPEN_P11951A]
X=102678.00714
Y=204134.926708603

[QTBDY__P11951q]
X=102678.00714
Y=205027.098572841

[SLUICE_VERTICAL_P11951U]
X=102678.00714
Y=205919.27043708

[RIVER_SECTION_P11951D]
X=102678.00714
Y=206811.442301319

[RIVER_SECTION_P13.000]
X=102678.00714
Y=207703.614165557

[RIVER_SECTION_P14.000]
X=102678.00714
Y=208595.786029796

[RIVER_SECTION_P15.000]
X=102678.00714
Y=209487.957894034

[RIVER_SECTION_P16.000]
X=102678.00714
Y=210380.129758273

[RIVER_SECTION_P17.000]
X=102678.00714
Y=211272.301622512

[RIVER_SECTION_P18.000]
X=102678.00714
Y=212164.47348675

[RIVER_SECTION_P19.000]
X=102678.00714
Y=213056.645350989

[RIVER_SECTION_P20.000]
X=102678.00714
Y=213948.817215228

[RIVER_SECTION_P21.000]
X=102678.00714
Y=214840.989079466

[RIVER_SECTION_P22.000]
X=102678.00714
Y=215733.160943705

[RIVER_SECTION_P23.000]
X=102678.00714
Y=216625.332807943

[RIVER_SECTION_P24.000]
X=102678.00714
Y=217517.504672182

[RIVER_SECTION_P25.000]
X=102678.00714
Y=218409.676536421

[RIVER_SECTION_P26.000]
X=102678.00714
Y=219301.848400659

[RIVER_SECTION_P27.000]
X=102678.00714
Y=220194.020264898

[RIVER_SECTION_P28.000]
X=102678.00714
Y=221086.192129137

[RIVER_SECTION_P29.000]
X=102678.00714
Y=221978.363993375

[RIVER_SECTION_P30.000]
X=102678.00714
Y=222870.535857614

[RIVER_SECTION_P31.000]
X=102678.00714
Y=223762.707721852

[RIVER_SECTION_P32.000]
X=102678.00714
Y=224654.879586091

[RIVER_SECTION_P33.000]
X=102678.00714
Y=225547.05145033

[RIVER_SECTION_P34.000]
X=102678.00714
Y=226439.223314568

[RIVER_SECTION_P35.000]
X=102678.00714
Y=227331.395178807

[RIVER_SECTION_P36.000]
X=102678.00714
Y=228223.567043046

[RIVER_SECTION_P37.000]
X=102678.00714
Y=229115.738907284

[RIVER_SECTION_P38.000]
X=102678.00714
Y=230007.910771523

[RIVER_SECTION_P39.000]
X=102678.00714
Y=230900.082635761

[RIVER_SECTION_P40.000]
X=102678.00714
Y=231792.2545

[RIVER_SECTION_P41.000]
X=102678.00714
Y=232684.426364239

[RIVER_SECTION_P42.000]
X=102678.00714
Y=233576.598228477

[RIVER_SECTION_P43.000]
X=102678.00714
Y=234468.770092716

[RIVER_SECTION_P44.000]
X=102678.00714
Y=235360.941956954

[RIVER_SECTION_P44865J]
X=102678.00714
Y=236253.113821193

[JUNCTION_OPEN_P44865J]
X=102678.00714
Y=237145.285685432

[QTBDY__P44865O]
X=102678.00714
Y=238037.45754967

[SLUICE_VERTICAL_P44865U]
X=102678.00714
Y=238929.629413909

[RIVER_SECTION_P44865D]
X=102678.00714
Y=239821.801278148

[RIVER_SECTION_P46.000]
X=102678.00714
Y=240713.973142386

[RIVER_SECTION_P47.000]
X=102678.00714
Y=241606.145006625

[RIVER_SECTION_P48.000]
X=102678.00714
Y=242498.316870863

[RIVER_SECTION_P49.000]
X=102678.00714
Y=243390.488735102

[RIVER_SECTION_P50.000]
X=102678.00714
Y=244282.660599341

[RIVER_SECTION_P51.000]
X=102678.00714
Y=245174.832463579

[RIVER_SECTION_P52.000]
X=102678.00714
Y=246067.004327818

[RIVER_SECTION_P53.000]
X=102678.00714
Y=246959.176192057

[RIVER_SECTION_P54.000]
X=102678.00714
Y=247851.348056295

[RIVER_SECTION_P55.000]
X=102678.00714
Y=248743.519920534

[RIVER_SECTION_P56.000]
X=102678.00714
Y=249635.691784772

[RIVER_SECTION_P57.000]
X=102678.00714
Y=250527.863649011

[RIVER_SECTION_P58.000]
X=102678.00714
Y=251420.03551325

[RIVER_SECTION_P59.000]
X=102678.00714
Y=252312.207377488

[RIVER_SECTION_P60.000]
X=102678.00714
Y=253204.379241727

[RIVER_SECTION_P61.000]
X=102678.00714
Y=254096.551105965

[RIVER_SECTION_P62.000]
X=102678.00714
Y=254988.722970204

[RIVER_SECTION_P63.000]
X=102678.00714
Y=255880.894834443

[RIVER_SECTION_P64.000]
X=102678.00714
Y=256773.066698681

[RIVER_SECTION_P65.000]
X=102678.00714
Y=257665.23856292

[RIVER_SECTION_P66.000]
X=102678.00714
Y=258557.410427159

[RIVER_SECTION_P67.000]
X=102678.00714
Y=259449.582291397

[RIVER_SECTION_P68.000]
X=102678.00714
Y=260341.754155636

[RIVER_SECTION_P69.000]
X=102678.00714
Y=261233.926019874

[RIVER_SECTION_P70.000]
X=102678.00714
Y=262126.097884113

[RIVER_SECTION_P71.000]
X=102678.00714
Y=263018.269748352

[RIVER_SECTION_P72.000]
X=102678.00714
Y=263910.44161259

[RIVER_SECTION_P73.000]
X=102678.00714
Y=264802.613476829

[RIVER_SECTION_P74.000]
X=102678.00714
Y=265694.785341067

[RIVER_SECTION_P75.000]
X=102678.00714
Y=266586.957205306

[RIVER_SECTION_P76.000]
X=102678.00714
Y=267479.129069545

[RIVER_SECTION_P77.000]
X=102678.00714
Y=268371.300933783

[RIVER_SECTION_P78.000]
X=102678.00714
Y=269263.472798022

[RIVER_SECTION_P79.000]
X=102678.00714
Y=270155.644662261

[RIVER_SECTION_P80.000]
X=102678.00714
Y=271047.816526499

[RIVER_SECTION_P81.000]
X=102678.00714
Y=271939.988390738

[RIVER_SECTION_P82.000]
X=102678.00714
Y=272832.160254976

[RIVER_SECTION_P83307]
X=102678.00714
Y=273724.332119215

[HTBDY__P83307]
X=102678.00714
Y=274616.503983454

[Connections]
ConnectionCount=96
1=QTBDY__P0000,JUNCTION_OPEN_P0000
2=JUNCTION_OPEN_P0000,SLUICE_VERTICAL_P0000a
3=JUNCTION_OPEN_P0000,RIVER_SECTION_P0000c
4=SLUICE_VERTICAL_P0000a,HTBDY__P0000b
5=RIVER_SECTION_P0000c,RIVER_SECTION_P1000
6=RIVER_SECTION_P1000,RIVER_SECTION_P2000
7=RIVER_SECTION_P2000,RIVER_SECTION_P3000
8=RIVER_SECTION_P3000,RIVER_SECTION_P4000
9=RIVER_SECTION_P4000,RIVER_SECTION_P5000
10=RIVER_SECTION_P5000,RIVER_SECTION_P6000
11=RIVER_SECTION_P6000,RIVER_SECTION_P7000
12=RIVER_SECTION_P7000,RIVER_SECTION_P8000
13=RIVER_SECTION_P8000,RIVER_SECTION_P9000
14=RIVER_SECTION_P9000,RIVER_SECTION_P10000
15=RIVER_SECTION_P10000,RIVER_SECTION_P11000
16=RIVER_SECTION_P11000,RIVER_SECTION_P11951A
17=RIVER_SECTION_P11951A,JUNCTION_OPEN_P11951A
18=JUNCTION_OPEN_P11951A,SLUICE_VERTICAL_P11951U
19=JUNCTION_OPEN_P11951A,QTBDY__P11951q
20=SLUICE_VERTICAL_P11951U,RIVER_SECTION_P11951D
21=RIVER_SECTION_P11951D,RIVER_SECTION_P13.000
22=RIVER_SECTION_P13.000,RIVER_SECTION_P14.000
23=RIVER_SECTION_P14.000,RIVER_SECTION_P15.000
24=RIVER_SECTION_P15.000,RIVER_SECTION_P16.000
25=RIVER_SECTION_P16.000,RIVER_SECTION_P17.000
26=RIVER_SECTION_P17.000,RIVER_SECTION_P18.000
27=RIVER_SECTION_P18.000,RIVER_SECTION_P19.000
28=RIVER_SECTION_P19.000,RIVER_SECTION_P20.000
29=RIVER_SECTION_P20.000,RIVER_SECTION_P21.000
30=RIVER_SECTION_P21.000,RIVER_SECTION_P22.000
31=RIVER_SECTION_P22.000,RIVER_SECTION_P23.000
32=RIVER_SECTION_P23.000,RIVER_SECTION_P24.000
33=RIVER_SECTION_P24.000,RIVER_SECTION_P25.000
34=RIVER_SECTION_P25.000,RIVER_SECTION_P26.000
35=RIVER_SECTION_P26.000,RIVER_SECTION_P27.000
36=RIVER_SECTION_P27.000,RIVER_SECTION_P28.000
37=RIVER_SECTION_P28.000,RIVER_SECTION_P29.000
38=RIVER_SECTION_P29.000,RIVER_SECTION_P30.000
39=RIVER_SECTION_P30.000,RIVER_SECTION_P31.000
40=RIVER_SECTION_P31.000,RIVER_SECTION_P32.000
41=RIVER_SECTION_P32.000,RIVER_SECTION_P33.000
42=RIVER_SECTION_P33.000,RIVER_SECTION_P34.000
43=RIVER_SECTION_P34.000,RIVER_SECTION_P35.000
44=RIVER_SECTION_P35.000,RIVER_SECTION_P36.000
45=RIVER_SECTION_P36.000,RIVER_SECTION_P37.000
46=RIVER_SECTION_P37.000,RIVER_SECTION_P38.000
47=RIVER_SECTION_P38.000,RIVER_SECTION_P39.000
48=RIVER_SECTION_P39.000,RIVER_SECTION_P40.000
49=RIVER_SECTION_P40.000,RIVER_SECTION_P41.000
50=RIVER_SECTION_P41.000,RIVER_SECTION_P42.000
51=RIVER_SECTION_P42.000,RIVER_SECTION_P43.000
52=RIVER_SECTION_P43.000,RIVER_SECTION_P44.000
53=RIVER_SECTION_P44.000,RIVER_SECTION_P44865J
54=RIVER_SECTION_P44865J,JUNCTION_OPEN_P44865J
55=JUNCTION_OPEN_P44865J,SLUICE_VERTICAL_P44865U
56=JUNCTION_OPEN_P44865J,QTBDY__P44865O
57=SLUICE_VERTICAL_P44865U,RIVER_SECTION_P44865D
58=RIVER_SECTION_P44865D,RIVER_SECTION_P46.000
59=RIVER_SECTION_P46.000,RIVER_SECTION_P47.000
60=RIVER_SECTION_P47.000,RIVER_SECTION_P48.000
61=RIVER_SECTION_P48.000,RIVER_SECTION_P49.000
62=RIVER_SECTION_P49.000,RIVER_SECTION_P50.000
63=RIVER_SECTION_P50.000,RIVER_SECTION_P51.000
64=RIVER_SECTION_P51.000,RIVER_SECTION_P52.000
65=RIVER_SECTION_P52.000,RIVER_SECTION_P53.000
66=RIVER_SECTION_P53.000,RIVER_SECTION_P54.000
67=RIVER_SECTION_P54.000,RIVER_SECTION_P55.000
68=RIVER_SECTION_P55.000,RIVER_SECTION_P56.000
69=RIVER_SECTION_P56.000,RIVER_SECTION_P57.000
70=RIVER_SECTION_P57.000,RIVER_SECTION_P58.000
71=RIVER_SECTION_P58.000,RIVER_SECTION_P59.000
72=RIVER_SECTION_P59.000,RIVER_SECTION_P60.000
73=RIVER_SECTION_P60.000,RIVER_SECTION_P61.000
74=RIVER_SECTION_P61.000,RIVER_SECTION_P62.000
75=RIVER_SECTION_P62.000,RIVER_SECTION_P63.000
76=RIVER_SECTION_P63.000,RIVER_SECTION_P64.000
77=RIVER_SECTION_P64.000,RIVER_SECTION_P65.000
78=RIVER_SECTION_P65.000,RIVER_SECTION_P66.000
79=RIVER_SECTION_P66.000,RIVER_SECTION_P67.000
80=RIVER_SECTION_P67.000,RIVER_SECTION_P68.000
81=RIVER_SECTION_P68.000,RIVER_SECTION_P69.000
82=RIVER_SECTION_P69.000,RIVER_SECTION_P70.000
83=RIVER_SECTION_P70.000,RIVER_SECTION_P71.000
84=RIVER_SECTION_P71.000,RIVER_SECTION_P72.000
85=RIVER_SECTION_P72.000,RIVER_SECTION_P73.000
86=RIVER_SECTION_P73.000,RIVER_SECTION_P74.000
87=RIVER_SECTION_P74.000,RIVER_SECTION_P75.000
88=RIVER_SECTION_P75.000,RIVER_SECTION_P76.000
89=RIVER_SECTION_P76.000,RIVER_SECTION_P77.000
90=RIVER_SECTION_P77.000,RIVER_SECTION_P78.000
91=RIVER_SECTION_P78.000,RIVER_SECTION_P79.000
92=RIVER_SECTION_P79.000,RIVER_SECTION_P80.000
93=RIVER_SECTION_P80.000,RIVER_SECTION_P81.000
94=RIVER_SECTION_P81.000,RIVER_SECTION_P82.000
95=RIVER_SECTION_P82.000,RIVER_SECTION_P83307
96=RIVER_SECTION_P83307,HTBDY__P83307

[VISTEXT]
VISTEXTCOUNT=0

[Font]
Font name=Tahoma
Charset=1
Height=-11
Bold=0
Italic=0
Underline=0
Strikeout=0

[Text]
Layout=1

[Display]
MiniNodes=0
MiniStructureNodes=0
NodeLabels=0
Connections=1

[General]
UnitMove=0
ExportConnections=0
GeneralConnectionsColour=0
LateralConnectionsColour=16711680
SpillConnectionsColour=8421504

[Grid]
Show=0
X=25
Y=25

[RWC]
Top=275508.675847692
Left=87355.83005
Right=240577.60095
Bottom=188075.833152308

[PC]
Top=0
Left=0
Right=0
Bottom=0

