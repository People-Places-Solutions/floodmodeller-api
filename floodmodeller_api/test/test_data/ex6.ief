[ISIS Event Header]
Title=NEW TEST TITLE
Datafile=EX6.DAT
Results=ex6
[ISIS Event Details]
RunType=Unsteady
InitialConditions=EX6.zzs
Start=0
Finish=8760
Timestep=1800
SaveInterval=86400
Sediment=1
ExtraTimesteps=0
RefineBridgeSecProps=0
SolveDHEqualsZeroAtStart=0
RulesAtTimeZero=0
RulesOnFirstIteration=1
ResetTimesAfterPos=0
UseFPSModularLimit=0
UseRemoteQ=0
New_Attribute=Wiggly
