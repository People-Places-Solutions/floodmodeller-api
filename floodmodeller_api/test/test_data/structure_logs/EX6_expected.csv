Unit Name,Unit Type,Unit Subtype,Comment,Friction,Dimensions (m),Weir Coefficient,Culvert Inlet/Outlet Loss
P0000a,SLUICE,VERTICAL,,,Crest Elevation: 73.20 x w: 20.00 x l: 1.00,,
P11951U,SLUICE,VERTICAL,,,Crest Elevation: 73.00 x w: 3.36 x l: 1.00,,
P44865U,SLUICE,VERTICAL,,,Crest Elevation: 70.81 x w: 3.36 x l: 3.00,,
